### DISCUSSION TITLE
Add a new tab in MainView

### DISCUSSION THREAD

**Discussion by macmanios** [kean/Pulse#118] [General]
Hi,
I would like to add a new custom tab on the MainView for SwiftUI
I have read the documentation on **https://kean-docs.github.io/pulseui/documentation/pulseui/** 
It looks like the guide got cut:

`In addition to [MainView](https://kean-docs.github.io/pulseui/documentation/pulseui/mainview), you can also show individual tabs. For example, if you have an existing debug menu and`

Could you tell me what public property to use to access the tab edition ?

**Comment by kean** [comment]
Starting with 3.0, the `MainView` was deprecated. The framework is now centered around a single screen: `ConsoleView` that has no tab bar, so you can add it to any navigation hierarchy.
